udata_front/theme/gouvfr/assets/js/pages/OrganizationPublishingForm/Step2DescribeOrganization.vue

```diff
@@ -275,14 +287,14 @@
   const checkOrga = ref({
     name: '',
     siren: '',
-    isPublicService: '',
+    isPublicService: false,
     exists: null as boolean | null
   });
 
   const checkBusinessId = () => {
-    if (organization.business_number_id.length == 0) {
+    if (!organization.business_number_id || organization.business_number_id?.length == 0) {
       return true;
-    } else if (organization.business_number_id.length == 14 && checkOrga.value.exists) {
+    } else if (organization.business_number_id?.length == 14 && checkOrga.value.exists) {
```

Comment:
It's weird you need the `?.` because you check for `!organization.business_number_id` at the beginning? A TypeScript bug/limitation?

Reply:
I don't recall, I will try without them. It's because we are still not using @datagouv/component package and the old one doesn't have `business_number_id` in the `Organization` type so the typechecking is not working.